cdap-docs/examples-manual/source/examples/index.rst

```diff
@@ -36,7 +36,10 @@ In addition to the :ref:`Getting Started's <getting-started-index>`
 
   * - Example Name
     - Description
-  * - :doc:`Hello World<hello-world>`
+  * - :doc:`Data Cleansing<data-cleansing>`
+    - TODO: A simple HelloWorld App that's written using CDAP. It introduces how the components stream, flow, dataset,
+      and service are used in a CDAP application.
+      * - :doc:`Hello World<hello-world>`
```

Comment:
This is messed up. Line 42 needs to be flush left; the TODO needs to go, and the description needs rewriting...Data Cleansing should go after "Count Random"; HelloWorld should probably be th first example, and then all examples after that can be alphabetical...